terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/scale-down.ts

```diff
@@ -290,7 +291,11 @@ export async function cleanupOldSSMParameters(runnersRegions: Set<string>, metri
   }
 }
 
-export async function getGHRunnerOrg(ec2runner: RunnerInfo, metrics: ScaleDownMetrics): Promise<GhRunner | undefined> {
+export async function getGHRunnerOrg(
+  ec2runner: RunnerInfo,
+  metrics: ScaleDownMetrics,
+  dontTrustIdleFromList = true,
```

Comment:
Perhaps a friendlier name :)

```suggestion
  verifyCachedIdleStatus = true,
```